script/install-as-user

```diff
@@ -127,6 +127,12 @@ sed -r \
     -e "s,# constraint_disabling: false,  constraint_disabling: false," \
     config/database.yml-example > config/database.yml
 
+# create the template_utf8 template we'll use for our databases
+createdb -T template0 -E UTF-8 template_utf8
+echo "update pg_database set datistemplate=true datallowconn=false where datname='template_utf8';" > /tmp/update-template.sql
+sudo -u postgres psql -f /tmp/update-template.sql
```

Comment:
This line will only run successfully if the user installing alaveteli has sudo rights/access. If sudo is needed, it needs to be in /site-specific-install.sh, not install-as-user. Worth a function in commonlib's install-site, if this might be useful elsewhere (similarly to the postgis function)?

(Also, you can run a command directly, no need for a temporary file here.)